video_xblock/constants.py

```diff
@@ -0,0 +1,27 @@
+"""
+Lists of constants that can be used in the video xblock.
+"""
+
+
+class status(object):  # pylint: disable=invalid-name
+    """
+    Contains HTTP codes used in video xblock.
+    """
+
+    HTTP_200_OK = 200
+    HTTP_201_CREATED = 201
+    HTTP_401_UNAUTHORIZED = 401
+    HTTP_404_NOT_FOUND = 404
```

Comment:
> We try to keep it consistent wit DRF status module

Do you use DRF in this module? What is the reason to duplicate something we already have?